prefq/query_client.py

```diff
@@ -41,6 +42,10 @@ def send_video_pair(self, query_id, left_filename, right_filename, video_dir):
                 json.dumps(query_id),
                 "application/json",
             ),
+            "file_extension": (
```

Comment:
We already send the filename, so isn't this redundant? You could argue that there is no reason to send the filename in the first place, only the extension. Either way, sending both seems unnecessary.

Reply:
good point, I've totally overseen the easiest solution! i adjusted changes on the server script, & created a branch/PR with a more appropriate name